sound/soc/sof/sof-priv.h

```diff
@@ -411,10 +411,15 @@ struct snd_sof_dev {
 	struct snd_dma_buffer dmatb;
 	struct snd_dma_buffer dmatp;
 	int dma_trace_pages;
-	wait_queue_head_t trace_sleep;
 	u32 host_offset;
-	u32 dtrace_is_enabled;
-	u32 dtrace_error;
+	unsigned int dtrace_is_enabled:1; /* FW DMA trace is active */
+	unsigned int dtrace_error:1;      /* FW sent non-recoverable error */
+
+	/* DFS trace client state */
+	wait_queue_head_t dftrace_sleep;
```

Comment:
Not sure I see the  point of renaming this? we have a mix of 'trace', 'dtrace', 'dftrace' and it starts to look messy?

Reply:
@plbossart Hmm, I wanted to split state we have towards FW ("DMA trace" -> dtrace) and state related to the debugfs trace node ("debugfs trace" -> dftrace). Now the two are a bit mixed in the code, making it harder to follow. But I take it did not help. I'll drop.